### PR TITLE
refactored if-else ladder and started? method in spark module definition

### PR DESCRIPTION
**Pull request by bitfurry**

### CODE REVIEW

**Comment by bitfurry** [comment]
@ondra-m Are you okay with commit message?

**Comment by bitfurry** [comment]
@ondra-m any comments please ?
I want to go ahead with more refactoring  and also will be try complete pendingTODO .

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Ruby-Spark [![Build Status](https://travis-ci.org/ondra-m/ruby-spark.svg?branch=master)](https://travis-ci.org/ondra-m/ruby-spark)
+# Ruby-Spark [![Build Status](https://travis-ci.org/bitfurry/ruby-spark.svg)](https://travis-ci.org/bitfurry/ruby-spark)
```
Comment (ondra-m) [inline README.md]: Why?

File: lib/spark.rb [hunk 2]
```diff
@@ -107,7 +105,7 @@ def self.stop
   end
 
   def self.started?
-    !!@context
+    @context
```
Comment (ondra-m) [inline lib/spark.rb]: `started?` should return true or false

File: README.md (hunk 1 above)
Comment (bitfurry) [inline README.md]: That is my mistake,Sorry. I changed it for my own purpose.

File: lib/spark.rb (hunk 2 above)
Comment (bitfurry) [inline lib/spark.rb]: '!!' is same as not of not which would always be same as  @context only.

Comment (ondra-m) [inline lib/spark.rb]: Not true.

```ruby
  @context # => Spark::Context
 !@context # => false
!!@context # => true
```

**Pull request closed without merging**